src/components/shared/TokenAmountInput/TokenAmountInput.tsx

```diff
@@ -86,7 +89,7 @@ const TokenAmountInput = ({ colony, disabled }: Props) => {
           <div className={styles.tokenAmountUsd}>
             <EthUsd
               // appearance={{ theme: 'grey' }}
-              value={values.amount}
+              value={values.amount.replace(/,/g, '') || 0} // @NOTE: Get the amount without commas as BigNumber doesn't like them
```

Comment:
I think the problem should be fixed at the root (`FormattedInputComponent` should be storing the Cleave's raw value instead of formatted value which may contain the comma separators). I fixed it in #320 so do you want to add a TODO comment here to change it later?

Reply:
For sure, I will add it to the other instances where we are removing commas like this as this ain't the only instance.